docs/Troubleshooting.md

```diff
@@ -228,7 +228,7 @@ This is a known issue an occurs on first upload of Linux data into an OMS worksp
 * If you see DSC resource "class not found" error in omsconfig.log, please run `sudo /opt/omi/bin/service_control restart`.
 * In some cases, when the OMS Agent for Linux cannot talk to the OMS Service, data on the Agent is backed up to the full buffer size: 50 MB. The OMS Agent for Linux should be restarted by running the following command `/opt/microsoft/omsagent/bin/service_control restart`.
  * **Note:** This issue is fixed in Agent version >= 1.1.0-28
-* Make sure the _cron_ service is running. You can use `service cron status` (Debian, Ubuntu, SUSE) or `service crond status` (RHEL, CentOS) to check the status of this service. If the service does not exist, you can install the binaries and start the service using the following:
+* If `omsconfig.log` log file does not indicate that `PerformRequiredConfigurationChecks` operations are running periodically on the system, there might be a problem with the _cron_ job/service. Make sure _cron_ job exists under `/etc/cron.d/OMSConsistencyInvoker`. Also, make sure the _cron_ service is running. You can use `service cron status` (Debian, Ubuntu, SUSE) or `service crond status` (RHEL, CentOS) to check the status of this service. If the service does not exist, you can install the binaries and start the service using the following:
```

Comment:
Looks good, except I see _ (underscore before and after the work cron can you remove that/

Reply:
That makes it _Italic_ and it won't show up in the actual document, but I'll remove it.